### DISCUSSION TITLE
PR Review with IntelliJ IDEA

### DISCUSSION THREAD

**Discussion by lebogg** [clouditor/clouditor#434] [Ideas]
In IntelliJ IDEA, there is the option to review PRs within the IDE (see also this [stack overflow answer](https://stackoverflow.com/a/58037092/16445647)). When I try to add this feature in the IDE, the following error message appears: "`clouditor` organization has enabled OAuth App access restrictions, meaning that data access to third-parties is limited".
-> Can you allow it or is it against some policy, @oxisto ?

**Comment by oxisto** [comment]
It is intentional to not allow all third-party apps by default. Normally, the application would register itself and I could manually approve it. For some reason, the IntelliJ one doesn't show up at all. I am not really in favour of removing the restriction alltogather.

**Reply by oxisto** [reply]
It is granted now.

**Comment by lebogg** [comment]
Thanks!
